### PR TITLE
Portenta X8 Out of the box tutorial fixes

### PR DESCRIPTION
**Pull request by martab1994**
Fix Cloud PRO subscription

## What This PR Changes
- (Please explain here why you created the pull request and specify what it changes)

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md [hunk 1]
```diff
@@ -24,7 +24,6 @@ software:
 
 * [Portenta X8](https://store.arduino.cc/products/portenta-x8) board.
 * Wi-Fi AP with Internet access.
-* Arduino Pro Cloud Subscription. [Learn more about the Pro Cloud](https://www.arduino.cc/pro/hardware/product/portenta-x8#pro-cloud).
```
Comment (marqdevx) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md:27]: Should not we add something pointing to that link somewhere else?

File: content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md [hunk 2]
```diff
@@ -59,16 +58,17 @@ Once it is connected, you should see the Wi-Fi status bullet in the bottom left
 
 ***You can change your network by clicking on the button again and repeat the above steps***
 
-## Connect to a FoundriesFactory
+## Connect to a Foundries Factory
```
Comment (marqdevx) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md:61]: @vanmaegima For consistency should we use it all together? in which cases can we use it with the space? 🤔

Comment (vanmaegima) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md:61]: Let's ask @kprosise about this

Comment (marqdevx) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md:61]: Clear! We will have this in mind. 
Thanks 😄

File: content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md [hunk 3]
```diff
@@ -108,7 +108,7 @@ Once it is completed, the Factory button on the Portenta X8 dashboard will turn
 
 ![Successful connection](assets/x8-oob-factory-success.png)
 
-#### Check Your FoundriesFactory
+#### Check Your Foundries Factory
```
Comment (marqdevx) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md:111]:
```suggestion
#### Check Your Factory
```

**Review by marqdevx — CHANGES_REQUESTED** [review]

File: content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md (hunk 1 above)
Comment (martab1994) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md:27]: It's already mentioned in the Register to Foundries Factory section :)

**Review by marqdevx — APPROVED** [review]
LGTM

**Pull request merged**